Assignment-1/services/src/parseinput.go

```diff
@@ -0,0 +1,61 @@
+package src
+
+import (
+	"bufio"
+	"fmt"
+	"nuclei-assignment-1/utils"
+	"os"
+	"strings"
+)
+
+func ParseInputName() string {
+	name := ""
+	reader := bufio.NewReader(os.Stdin)
+	InputNameDetails()
+	for name == "" {
+		text, _ := reader.ReadString('\n')
+		name, _ = utils.ValidateName(text)
+	}
+	return name
+}
+
+func ParseInputString() (string, float64, int) {
+	itype, iprice, iquantity := "", 0., 0
+	InputDetails()
+	reader := bufio.NewReader(os.Stdin)
+naming:
+	for itype == "" || iprice == 0. || iquantity == 0 {
+		text, _ := reader.ReadString('\n')
```

Comment:
don't ignore error. && use fmt.scan

Reply:
fmt.Scan return int and err which is not required as here the requirement is of string value, as the order in which the other three inputs will come is not defined so i have to take them as string and later convert it.
fixed the error handling part